src/platforms/javascript/common/install/loader.mdx

```diff
@@ -0,0 +1,234 @@
+
+```html
+<script
+  src="https://js.sentry-cdn.com/___PUBLIC_KEY___.min.js"
+  crossorigin="anonymous"
+></script>
+```
+
+By default, Performance Monitoring and Session Replay is enabled. 
+
+## Source Maps
+
+In order to have working SourceMaps when using the Loader Script, you will have to [upload & host them publicly](/platforms/javascript/sourcemaps/uploading/hosting-publicly/).
+
+## Loader Configuration
+
+The loader has a few configuration options:
+
+- What version of the SDK to load
+- Using Performance Monitoring
+- Using Session Replay
+- Show debug logs
+
+### SDK Version
+
+To configure the version, use the dropdown in the _JavaScript Loader_ settings, directly beneath the script tag you copied earlier.
+
+![JavaScript Loader Settings](js-loader-settings.png)
+
+Note that because of caching, it can take a few minutes for version changes made here to take effect.
+
+### Load Timing
+
+If you only use the Loader for errors, the loader won't load the full SDK until triggered by one of the following:
+
+- an unhandled error
+- an unhandled promise rejection
+- a call to `Sentry.captureException`
+- a call to `Sentry.captureMessage`
+- a call to `Sentry.captureEvent`
+
+Once one of those occurs, the loader will buffer that event and immediately request the full SDK from our CDN. Any events that occur between that request being made and the completion of SDK initialization will also be buffered, and all buffered events will be sent to Sentry once the SDK is fully initialized.
+
+Alternatively, you can set the loader to request the full SDK earlier: still as part of page load, but _after_ all of the other JavaScript on the page has run. (In other words, in a subsequent event loop.) To do this, include `data-lazy="no"` in your script tag.
+
+```html
+<script
+  src="https://js.sentry-cdn.com/___PUBLIC_KEY___.min.js"
+  crossorigin="anonymous"
+  data-lazy="no"
+></script>
+```
+
+Finally, if you want to control the timing yourself, you can call `Sentry.forceLoad()`. You can do this as early as immediately after the loader runs (which has the same effect as setting `data-lazy="no"`) and as late as the first unhandled error, unhandled promise rejection, or call to `Sentry.captureMessage` or `Sentry.captureEvent` (which has the same effect as not calling it at all). Note that you can't delay loading past one of the aforementioned triggering events.
+
+If Performance Monitoring and/or Session Replay is enabled, we immediately need to fetch and initialize the bundle to make sure we are able to capture transactions/replays when the page loaded.
```

Comment:
```suggestion
If Performance Monitoring and/or Session Replay is enabled, the SDK will immediately fetch and initialize the bundle to make sure it can capture transactions and/or replays once the page loads.
```